src/main/java/module-info.java

```diff
@@ -0,0 +1,16 @@
+import ch.qos.logback.classic.spi.Configurator;
+import org.cryptomator.cli.LogbackConfigurator;
+
+open module org.cryptomator.cli {
+    uses Configurator;
```

Comment:
do we really "use" Configurator? Accessing it via its `INSTANCE` does not require ServiceLoading

Reply:
Fixed in https://github.com/cryptomator/cli/pull/68/commits/2e5b27141882274e60791698bd012ce0d63ab505